tests/machine.test.js

```diff
@@ -1134,4 +1134,177 @@ describe(OlmMachine.name, () => {
             expect(progressListener).toHaveBeenCalledWith(0, 1);
         });
     });
+
+    describe("Request missing secrets", () => {
+        async function getHypotheticalKeyQuery(new_user, new_device, bootstrapCrossSigning) {
+            let initialMachine = await OlmMachine.initialize(new_user, new_device);
+            const userId = initialMachine.userId.toString();
+            const deviceId = initialMachine.deviceId.toString();
+
+            let deviceKeys;
+            let outgoingRequests = await initialMachine.outgoingRequests();
+            for (const request of outgoingRequests) {
+                if (request instanceof KeysUploadRequest) {
+                    deviceKeys = JSON.parse(request.body);
+                }
+            }
+            delete deviceKeys.one_time_keys;
+            delete deviceKeys.fallback_keys;
+
+            let xsigning;
```

Comment:
we don't really use this abbreviation anywhere else. Suggest calling it `crossSigning` and sucking up the extra 4 characters.